draft-ietf-tls-tls13.md

```diff
@@ -1566,7 +1566,9 @@ cipher_suites
   order of client preference. If the list contains cipher suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
-  usual. Values are defined in {{cipher-suites}}.
+  usual. Values are defined in {{cipher-suites}}. If the client is
+  attempting a PSK key establishment, it MUST only advertise the single
+  cipher suite associated with the PSK.
```

Comment:
@davidben: true, only the hash part needs to be the one associated to the PSK.

@ekr: from the spec: "Prior to accepting PSK key establishment, the server MUST validate the corresponding binder value (see Section 4.2.6.1 below). If this value is not present or does not validate, the server MUST abort the handshake."